### ISSUE TITLE
incompatibility with SciPy 1.14 (removal of scipy.integrate.trapz)

### ISSUE THREAD

**Issue by slayoo** [bsumlin/PyMieScatt#25]
Due to removal of `scipy.integrate.trapz` (https://github.com/scipy/scipy/pull/20278) from SciPy 1.14.0, trying to use PyMieScatt fails at import with:
```
...site-packages\PyMieScatt\Mie.py:5
      3 import numpy as np
      4 from scipy.special import jv, yv
----> 5 from scipy.integrate import trapz
      6 import warnings
      8 def coerceDType(d):

ImportError: cannot import name 'trapz' from 'scipy.integrate' (...site-packages\scipy\integrate\__init__.py)
```

**Comment by natsuwater** [comment]
Hi, 

I've made a pull request on this.  https://github.com/bsumlin/PyMieScatt/pull/26

**Comment by slayoo** [comment]
Thanks!

**Comment by joelager** [comment]
Was able to fix this locally by replacing 'trapz' with 'trapezoid' in Mie.py and Inverse.py

**Comment by slayoo** [comment]
this was addressed in #26
